fyde-access-proxy/install-bm-vm.md

```diff
@@ -48,7 +48,7 @@ nav_order: 1
     Example for unattended instalation with Fyde Access Proxy token:
       - Specify the Fyde Access Proxy token inside quotes
 
-      ./install-fyde-proxy-linux.sh -p 443 -t "https://xxxxxxxxxxxx"
+      ./install-fyde-proxy-linux.sh -p 443 -t "https://xxxxxxxxxxxx" -u
```

Comment:
@GMartinez-Sisti what is the `-u` for?

Reply:
-u is unattended. The logic needs to be improved, but the idea is that if you specify that parameter it won't ask you for anything. Since there are situations where the port and key can be obtained via env or aws ssm/secret, this make sit explicit that we won't provide the parameters.